packages/modelviewer.dev/data/examples.json

```diff
@@ -31,19 +31,19 @@
     "examples": [
       {
         "htmlId": "demo-container-1",
-        "name": "Autoplay animation"
+        "name": "Autoplay"
       },
       {
         "htmlId": "demo-container-2",
-        "name": "Select specific"
+        "name": "Selecting Animations"
       },
       {
         "htmlId": "demo-container-3",
-        "name": "Animation crossfade"
+        "name": "Crossfade"
       },
       {
-        "htmlId": "demo-container-3",
-        "name": "Paused Model"
+        "htmlId": "demo-container-4",
```

Comment:
I would change the order to `Type`, `Options`, `Default Value`.

Reply:
done